### DISCUSSION TITLE
Textures not displaying properly after suspend on linux

### DISCUSSION THREAD

**Discussion by Cyliann** [ppy/osu#23301] [Q&A]
### Type

Cosmetic

### Bug description

If I leave a game open, suspend and wake up from suspend all the textures are not loading/displaying correctly.

Specs: Arch Linux X11 Nvidia (proprietary drivers)

It's not a new bug. It's there since I remember it.

### Screenshots or videos

https://user-images.githubusercontent.com/33868960/233857471-c09cd89e-57f9-4ae2-8619-952e21fb97dc.mp4

### Version

2023.419-0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/11304524/database.log)
[input.log](https://github.com/ppy/osu/files/11304525/input.log)
[network.log](https://github.com/ppy/osu/files/11304526/network.log)
[performance.log](https://github.com/ppy/osu/files/11304527/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/11304528/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/11304529/performance-draw.log)
[performance-update.log](https://github.com/ppy/osu/files/11304530/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/11304531/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11304532/updater.log)

**Comment by frenzibyte** [comment]
Likely GPU/driver issue, moving to discussions at least.

**Reply by frenzibyte** [reply]
Relevant read: https://askubuntu.com/questions/885369/broken-textures-after-resuming-from-sleep
